features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/data/Application.jsx

```diff
@@ -63,10 +63,14 @@ export default class Application extends Resource {
      */
     getKeys(keyType) {
         const promiseKeys = this.client.then((client) => {
-            return client.apis.Applications.get_applications__applicationId__keys({ applicationId: this.id });
+            console.log(client.apis['Application Keys']);
+
```

Comment:
remove the new line.

Reply:
this issue no longer exists, 

https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/data/Application.jsx